### PR TITLE
Week 1 model exercise

### PR DESCRIPTION
**Pull request by CYJW**
## Jumpstart Modules
Which week are you submitting completed practice exercises for? 

- [X] Week 1 (Models)
- [ ] Week 2 (Tests, Docs, Sources)
- [ ] Week 3 (Jinja, Macros, Packages)
- [ ] Week 4 (Materializations, Analyses, Seeds)
## Focus the feedback
What models/macros/files would you like feedback on?  Be specific.
```
SELECT SUM(LIFETIME_VALUE)
FROM "ANALYTICS"."DBT_JWATT"."DIM_CUSTOMERS"
```
gives me this result:  $167,200

### CODE REVIEW

**Comment by CYJW** [comment]
@amychen1776 
Please review my code.
Thanks

File: dbt_project.yml [hunk 1]
```diff
@@ -32,7 +32,9 @@ clean-targets:         # directories to be removed by `dbt clean`
...
-  my_new_project:
-      # Applies to all files under models/example/
-      example:
-          materialized: view
+  jaffle_shop:
+    marts:
+        core: 
+            +materialized: table
+    staging:
+        +materialzed: view
```
Comment (amychen1776) [inline dbt_project.yml:40]: Just a FYI, dbt actually defaults to materializing models as views so there is no need to explicitly config any models as views.

File: models/staging/stripe/stg_payments.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+SELECT
```
Comment (amychen1776) [inline models/staging/stripe/stg_payments.sql:1]: To conform to one SQL style, I recommend making this model all in lowercase.

**Review by amychen1776 — APPROVED** [review]
This looks great! I left some minor feedback but nothing to prevent the merge. :)

**Pull request merged**